package.json

```diff
@@ -183,7 +183,7 @@
     "abi-decoder": "^2.4.0",
     "axios": "0.21.4",
     "bignumber.js": "9.0.1",
-    "bnc-onboard": "~1.34.2",
+    "bnc-onboard": "1.34.1",
```

Comment:
Could someone update this to 1.34.2 for me? 😄 

I needed to update the lockfile but https://github.com/blocknative/onboard/issues/691